src/client/pages/OfferNew/Introduction/HeroOfferDetails.tsx

```diff
@@ -68,7 +68,8 @@ const Address = styled.div`
   display: inline-block;
 
   ${MEDIUM_SCREEN_MEDIA_QUERY} {
-    font-size: 2.5rem;
+    font-size: 3rem;
+    line-height: 3.5rem;
```

Comment:
Should we prefer unit-less line heights?

```suggestion
    line-height: 1.17;
```

> In most cases, this is the preferred way to set line-height and avoid unexpected results due to inheritance.

https://developer.mozilla.org/en-US/docs/Web/CSS/line-height

I also find it easier to read these values. They should also apply even across breakpoints so we can set it ones per component.

Reply:
@robinandeer changed the entire file to be according.